libsolidity/interface/CompilerStack.cpp

```diff
@@ -226,6 +233,7 @@ bool CompilerStack::analyze()
 
 		if (noErrors)
 		{
+			// Control flow graph: currently checks that variable is not used before it is assigned to.
```

Comment:
But only for storage variables. Also I would not say what it currently does, because we almost certainly forget to update that, but rather that it for example does this.

Reply:
Can you give a proposed example text?